vault/tao/hubs.md

```diff
@@ -0,0 +1,55 @@
+# Hubs
```

Comment:
If you do have a `hubs` directory you probably want this file to be at `hubs/index.md`. both will work but that is better IMO.

Reply:
Actioned: https://github.com/life-itself/community/pull/202/commits/796e1c9f27751ba2e28aa3726366e9a4dbba1939